test_conformance/api/test_queue_properties_queries.cpp

```diff
@@ -267,3 +267,79 @@ int test_queue_properties_queries(cl_device_id deviceID, cl_context context,
     }
     return error;
 }
+
+int test_set_command_queue_property(cl_device_id deviceID, cl_context context,
+                                    cl_command_queue queue, int num_elements)
+{
+    int err;
+
+    // Required minimum command queue properties
+    std::vector<cl_command_queue_properties> queue_property_options = {
+        0, CL_QUEUE_PROFILING_ENABLE
+    };
+
+    // Add other supported properties combinations
+    cl_command_queue_properties supported_queue_props;
+    clGetDeviceInfo(deviceID, CL_DEVICE_QUEUE_PROPERTIES,
+                    sizeof(supported_queue_props), &supported_queue_props,
+                    NULL);
+    if (supported_queue_props & CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE)
+    {
+        queue_property_options.push_back(
+            CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE);
+        queue_property_options.push_back(
```

Comment:
It does not make sense. First you build property only with `CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE` and in next line again `CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE` and `CL_QUEUE_PROFILING_ENABLE`